src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestUap.java

```diff
@@ -24,15 +24,16 @@
   private static final String MOBILE_IN_CONTENT = "#incontent_boxad_1";
   private static final String RESOLVED_STATE = "resolved_state=0";
   private static final String ARTICLE_MIDDLE = "#ArticleMidSection, #Header";
+  private static final String ARTICLE_FOOTER = ".article-footer";
 
   @Test(dataProviderClass = AdsDataProvider.class, dataProvider = "adsUapOasis", groups = "AdsUapOasis")
   public void adsUapOasis(
       Page page, List<Map<String, Object>> atfSlots, List<Map<String, Object>> btfSlots
   ) {
     AdsBaseObject ads = new AdsBaseObject(driver, page.getUrl(RESOLVED_STATE), WindowSize.DESKTOP);
-    verifySlotsUnblocked(ads, atfSlots);
+    verifySlotsUnblocked(ads, atfSlots, false);
     verifySlotsBlocked(ads, btfSlots);
-    verifySlotsUnblocked(ads, ListUtils.union(atfSlots, btfSlots));
+    verifySlotsUnblocked(ads, ListUtils.union(atfSlots, btfSlots), false);
   }
 
   @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5_DEFAULT // FIXME: use default emulator if mobile flag will be reverted
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1134)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)